src/styles/card.css

```diff
@@ -0,0 +1,114 @@
+.card {
+  display: block;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__logo {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  display: flex;
+  margin-bottom: 4px;
+}
+
+.card__link {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  text-decoration: none;
+  color: #060b35;
+}
+
+.card__code {
+  display: block;
+  margin-bottom: 16px;
+
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__value {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+
+  align-items: center;
+}
+
+.card__review {
+  display: inline-block;
+
+  align-self: center;
+
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.card__price {
+  display: inline-block;
+  padding-right: 85px;
+
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__cost {
+  display: inline-block;
+
+  font-family: Roboto, sans-serif;
```

Comment:
I think we don't nedd display inline block, its not working anyway, and font-family I put to html with class page and move it to own block. page.css